fanuc_lrmate200id_moveit_plugins/lrmate200id_kinematics/src/fanuc_lrmate200id_manipulator_ikfast_moveit_plugin.cpp

```diff
@@ -299,12 +299,12 @@ bool IKFastKinematicsPlugin::initialize(const std::string &robot_description,
 
   ROS_DEBUG_STREAM_NAMED("ikfast","Reading joints and links from URDF");
 
-  boost::shared_ptr<urdf::Link> link = boost::const_pointer_cast<urdf::Link>(robot_model.getLink(tip_frame_));
+  urdf::LinkSharedPtr link = urdf::const_pointer_cast<urdf::Link>(robot_model.getLink(tip_frame_));
```

Comment:
In ros-industrial/fanuc#262 the `Const` variants of the `*SharedPtr`s were used where appropriate ([here](https://github.com/ros-industrial/fanuc/blob/75a6f7045c87e3c4394407b39a3bdf7b8902833f/fanuc_lrmate200i_moveit_plugins/lrmate200i_kinematics/src/fanuc_lrmate200i_manipulator_ikfast_moveit_plugin.cpp#L302) fi).

Would be nice to do that here as well.